Code/GraphMol/SubstructLibrary/SubstructLibrary.h

```diff
@@ -559,7 +568,8 @@ class RDKIT_SUBSTRUCTLIBRARY_EXPORT SubstructLibrary {
   boost::shared_ptr<ROMol> operator[](unsigned int idx) {
     // expects implementation to throw IndexError if out of range
     PRECONDITION(mols, "molholder is null in SubstructLibrary");
-    return mols->getMol(idx);
+    const bool sanitize=true;
```

Comment:
Isn't setting `sanitize` to true here changing the current behavior?
There aren't that many circumstances where it's necessary, so I disagree with it being the default.

Reply:
This is the user facing API.  I think when the user calls getMol they should always get sanitized molecules.

Reply:
I'm not sure I agree with that, particularly since you aren't actually sanitizing the molecules - there are a number of properties that sanitization sets that won't be set here.
If it's in the end-user API and called "sanitize" then it should make sure you get back the equivalent of a molecule that's actually been sanitized.
I would prefer to just return the molecule in the state that it was being used during the search